shadows/framework/src/main/java/org/robolectric/shadows/ShadowNetworkCapabilities.java

```diff
@@ -54,6 +57,28 @@ public NetworkCapabilities removeCapability(int capability) {
         .removeCapability(capability);
   }
 
+  /**
+   * Changes {@link NetworkSpecifier} for this network capabilities. Works only on Android O and
+   * higher. For lower versions use {@link #setNetworkSpecifier(String)}
+   */
+  @HiddenApi
```

Comment:
nit: remove the `@HiddenApi` annotation (and below as well), they have no effect in Robolectric

Reply:
fixed